packages/color-wheel/README.md

```diff
@@ -1,64 +1,286 @@
 
-```
+```bash
 yarn add @spectrum-web-components/color-wheel
 ```
 
 Import the side effectful registration of `<sp-color-wheel>` via:
 
-```
+```javascript
 import '@spectrum-web-components/color-wheel/sp-color-wheel.js';
 ```
 
 When looking to leverage the `ColorWheel` base class as a type and/or for extension purposes, do so via:
 
-```
+```javascript
 import { ColorWheel } from '@spectrum-web-components/color-wheel';
 ```
 
-## Color Formatting
+## Anatomy
 
-When using the color elements, use `el.color` to access the `color` property, which should manage itself in the color format supplied. If you supply a color in `rgb()` format, `el.color` should return the color in `rgb()` format, as well.
+The color wheel consists of several key parts:
 
-The current color formats supported are as follows:
+- A circular track displaying the full spectrum of hues
+- A draggable handle that indicates the current hue selection
+- Accessible labels for screen readers
+- Optional custom gradient slot for advanced styling
 
-- Hex3, Hex4, Hex6, Hex8
-- HSV, HSVA
-- HSL, HSLA
-- RGB, RGBA
-- Strings (eg "red", "blue")
+```html
+<sp-color-wheel></sp-color-wheel>
+```
 
-**Please note for the following formats: HSV, HSVA, HSL, HSLA**
-When using the HSL or HSV formats, and a color's value (in HSV) is set to 0, or its luminosity (in HSL) is set to 0 or 1, the hue and saturation values may not be preserved by the element's `color` property. This is detailed in the [colorjs documentation](https://colorjs.io/docs/). Seperately, the element's `value` property is directly managed by the hue as represented in the interface.
+## Custom Gradient
+
+**⚠️ Deprecated Feature**
+
+The custom gradient functionality has been deprecated and is no longer supported. While the `gradient` slot may still be present in the component's code, it is broken and will not work as intended.
+
+If you previously relied on custom gradients for the color wheel, you should:
+
+- Use the default color wheel appearance
+- Consider alternative approaches for custom styling
+- Remove any existing custom gradient implementations
+
+**Note**: Even if you find the `gradient` slot in the component's source code, this feature is non-functional and should not be used in new implementations.
+
+## Options
+
+### Properties
+
+The color wheel supports several properties for configuration:
+
+#### Value (Hue)
 
-## Example
+Control the hue value directly (0-360 degrees):
 
 ```html
-<sp-color-wheel></sp-color-wheel>
+<sp-color-wheel value="180"></sp-color-wheel>
 ```
 
-### Disabled
+#### Color Values
 
-A color wheel in a disabled state shows that an input exists, but is not available in that circumstance. This can be used to maintain layout continuity and communicate that the wheel may become available later.
+The color wheel supports a wide variety of color formats for setting and getting color values:
```

Comment:
Can we add an example of this property above or below the color values table? To be be consistent with our sections. We have examples in all other sections of Options.